tx_pool/src/pool.rs

```diff
@@ -159,6 +159,8 @@ impl Pool {
         block_gas_limit: u64,
         account_gas_limit: AccountGasLimit,
         check_quota: bool,
+        emergency_brake: bool,
+        admin_address: Address,
```

Comment:
What about only one param `option<admin>` instead of `emergency_brake` and `admin_address` ? The same as protobuf.

Reply:
Protobuf does not have an option field, it will write the unset value as a default value. This time it is a bit awkward. You don't know if this all-zero address needs to be judged, so I wrote a bool value.

Reply:
Oh, I have already thought of a solution. The default bytes is an empty byte. It cannot be converted to an address. I can use the `option <admin>` directly. I will change it right away.